lib/auth.js

```diff
@@ -407,7 +244,9 @@ api.createIdentity = function(actor, identity, callback) {
           return callback(err);
         }
         // return unhashed passcode in record
-        record.identity.sysPasscode = results.generatePasscode;
+        // FIXME: is this necessary.  If so, a new solution will need to be
+        // implemented since passcode generation is happening elsewhere.
+        // record.identity.sysPasscode = results.generatePasscode;
```

Comment:
I think this was needed for the frontend or for an email that gets sent when the identity is created?